components/datePicker/datePicker.scss

```diff
@@ -0,0 +1,28 @@
+.ui-datepicker {
```

Comment:
because it doesn't use any SCSS features and later it will be easier to move from SCSS to CSS Modules